src/models/ProfileToFormRPartAInitialValues.ts

```diff
@@ -22,9 +22,12 @@ export function ProfileToFormRPartAInitialValues(
     programme && programme.curricula.length > 0
       ? programme.curricula
           .filter(c => c.curriculumSubType === MEDICAL_CURRICULUM)
-          .reduce(function (a, b) {
-            return a.curriculumStartDate > b.curriculumStartDate ? a : b;
-          })
+          .sort(
+            (a, b) =>
+              new Date(b.curriculumStartDate).getTime() -
+              new Date(a.curriculumStartDate).getTime()
```

Comment:
Unsure why I needed to initialize a new date here, but `curriculum.curriculumStartDate.getTime()` errored.

Reply:
It's JS magic.
Or it's because initialising a **new Date()** object gives us access to the getTime method while **Date()** gives us a string and so only those methods available to strings.
JS magic sounds better though.